cal/__init__.py

```diff
@@ -3,13 +3,45 @@
 
 from schema import db, Event, User
 from cal.fb import update_fb_events
+from celery import Celery
+
+def make_celery(app):
+    # Create the celery app.
+    celery = Celery(app.import_name, broker=app.config['CELERY_BROKER_URL'])
+    celery.conf.update(app.config)
+    TaskBase = celery.Task
+    class ContextTask(TaskBase):
+        abstract = True
+        def __call__(self, *args, **kwargs):
+            with app.app_context():
+                return TaskBase.__call__(self, *args, **kwargs)
+    celery.Task = ContextTask
+    return celery
+
+def create_app():
+    # Create the Flask app.
+    app = Flask(__name__)
+    app.config.from_object('config')
+    db.init_app(app)
+    whooshalchemy.whoosh_index(app, Event)
+    return app
+
+app = create_app()
+celery = make_celery(app)
+
+# Background task
+@celery.task(name="cal.fb_task")
+def fb_task_test():
+    print "Starting FB update"
+    update_fb_events()
+    print "Finished FB update"
+    # Update every 30 minutes
+    fb_task_test.apply_async(countdown=1800)
```

Comment:
Right this was mainly for debugging, when I couldn't get the task to start properly. I'll remove them for now.